mkdocs.yml

```diff
@@ -14,17 +14,12 @@ nav:
       - guides/openshock-how-to-flash-your-board.md
       - guides/openshock-first-setup.md
       - guides/openshock-how-to-update.md
-    - Shocklink:
```

Comment:
Maybe this should be OpenShock App and the previous category can be OpenShock Hardware?

Reply:
doesn't really makes sense to me

Reply:
This is not needed, as the guides in the openshock and shocklink groups both cover frontend functionality, think its ok to merge them